packages/core/src/command/hl7-notification/hl7-notification-webhook-sender-direct.ts

```diff
@@ -146,17 +147,23 @@ export class Hl7NotificationWebhookSenderDirect implements Hl7NotificationWebhoo
     const encounterPeriod = getEncounterPeriod(message);
     const encounterClass = getEncounterClass(message);
     const facilityName = getFacilityName(message);
-
-    analytics({
-      distinctId: cxId,
-      event: EventTypes.hl7NotificationReceived,
-      properties: {
-        cxId,
-        patientId,
-        messageCode,
-        triggerEvent,
-      },
-    });
+    const posthogApiKeyArn = Config.getPostHogApiKey();
+    if (posthogApiKeyArn) {
```

Comment:
should this ever not exist? should we consider throwing an error if the thing doesn't exist, instead of silently not sending mentrics? If we're doing this because we don't want to be sending metrics on develop, maybe we use that as the conditional instead